Dockerfile.dredd

```diff
@@ -6,6 +6,7 @@ RUN apk add --update --virtual build-dependencies \
       g++ \
       python \
       git \
+      npm \
```

Comment:
What? Is it not in the `nodejs` package installed 2 lines below? :thinking:

Reply:
If I'm correct, npm depends on nodejs, but nodejs doesn't depend on npm:
https://pkgs.alpinelinux.org/package/edge/main/x86_64/nodejs

Like rubygems depends on ruby, but not the opposite. Some OS/distros regroup
them in a single package because ruby upstream distribution includes rubygems,
but this isn't always practical, having separates ports/packages is more
convenient.

Reply:
I didn't knew about this difference, but I agree with you @tjouan:
On url you shared:
- nodejs is required by npm, but don't depends of npm
- npm only depends of nodejs-current

I didn't knew about, but adding adding `npm` to the list of dependencies does  fix error `/bin/sh: npm: not found`

Reply:
LGTM, I thought `npm` was packaged with `nodejs` :confused: